src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -389,8 +390,9 @@ bool emberAfGroupsClusterRemoveAllGroupsCallback(chip::app::Command * commandObj
                 }
                 else
                 {
-                    uint8_t groupName[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH + 1] = { 0 };
-                    GroupId groupId                                               = binding.groupId;
+                    GroupId groupId                                                 = binding.groupId;
+                    uint8_t groupNameBuffer[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH] = { 0 };
+                    chip::ByteSpan groupName(groupNameBuffer, sizeof(groupNameBuffer));
```

Comment:
```suggestion
                    chip::ByteSpan groupName(groupNameBuffer);
```
And it will infer the sizeof() bit from the type.